# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/Types.h
# Commit: 7b82837e2f56
# Full Hash: 7b82837e2f56582e1e86897285f4109856b72000
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-11-11 15:25:44
# Regressor Bug: 1739448
# File Overlap Count: 1
# Description:
#   Bug 1739448 - Implement a prototype WebGL-backed Canvas2D. r=gfx-reviewers,nical,jrmuizel
#   
#   This mainly provides DrawTargetWebgl, which implements the subset of the DrawTarget
#   API necessary for integration with CanvasRenderingContext2D. It translates them to
#   suitable commands for its internal ClientWebGLContext, which then manages remoting
# ==============================================================================

diff -r feac99f1dae3 -r 7b82837e2f56 gfx/2d/Types.h
--- a/gfx/2d/Types.h	Thu Nov 11 04:33:50 2021 +0000
+++ b/gfx/2d/Types.h	Thu Nov 11 04:33:50 2021 +0000
@@ -550,6 +550,7 @@
   RECORDING,
   DIRECT2D1_1,
   WEBRENDER_TEXT,
+  WEBGL,
 
   // Add new entries above this line.
   BACKEND_LAST
@@ -569,7 +570,8 @@
   CAIRO_CONTEXT,
   CGCONTEXT,
   CGCONTEXT_ACCELERATED,
-  OPENGL_TEXTURE
+  OPENGL_TEXTURE,
+  WEBGL_CONTEXT
 };
 
 enum class FontStyle : int8_t { NORMAL, ITALIC, BOLD, BOLD_ITALIC };